File: 02-week-2--recursion/01-day-1--recursive-counting/ruby/recursive_count.rb
def recursive_count(num = 0)
  return if num > 9
  puts num
  num += 1
  recursive_count(num)
end

if __FILE__ == $PROGRAM_NAME
  recursive_count
end

# OPTIONAL
# Please add your pseudocode to this file
# And a written explanation of your solution

File: 02-week-2--recursion/01-day-1--recursive-counting/javascript/recursive_count.js
function recursiveCount(num = 0) {
  if (num > 9) return
  console.log(num);
  num++;
  recursiveCount(num)
}

if (require.main === module) {
  recursiveCount();
}

module.exports = recursiveCount;

// OPTIONAL
// Please add your pseudocode to this file
// And a written explanation of your solution
